validation/src/main/resources/application.conf

```diff
@@ -158,6 +158,11 @@ hmda.validation.macro {
   Q065 {
     hoepaLoanLimit = 200
   }
+  Q071 {
+    currentYearThreshold = 50
```

Comment:
Those are test values. I'm planning to change them before I PR. Hence the "WIP:".